doc/user-guide.md

```diff
@@ -423,6 +423,62 @@ To implement complex deletion logic, you can add a finalizer to your Custom Reso
 deleted until you remove the finalizer (ie, after your cleanup logic has successfully run). For more information, see the
 [official Kubernetes documentation on finalizers](https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/#finalizers).
 
+**Example:**
+
+Following is a snippet from the controller file under `pkg/controller/memcached/memcached_controller.go`
+
+```go
+func (r *ReconcileMemcached) Reconcile(request reconcile.Request) (reconcile.Result, error) {
+    reqLogger := log.WithValues("Request.Namespace", request.Namespace, "Request.Name", request.Name)
+    reqLogger.Info("Reconciling Memcached")
+        
+    // Fetch the Memcached instance
+    memcached := &cachev1alpha1.Memcached{}
+    err := r.client.Get(context.TODO(), request.NamespacedName, memcached)
```

Comment:
Nit: Add the error check.
```Go
if err != nil {
	return reconcile.Result{}, err
}
```

Or, since this is just an example snippet, and we want to keep it short just add an ellipsis instead of the error check:
```Go
err := r.client.Get(context.TODO(), request.NamespacedName, memcached)
...
```

Reply:
Done